test/trans/Tests/NormalizeSpec.hs

```diff
@@ -0,0 +1,107 @@
+{-# LANGUAGE QuasiQuotes #-}
+-- | Sanity check test suite for equivalent and unequivalent programs.
```

Comment:
This entire test suite clarifies what it means for programs to be equivalent normalization (i.e., mangling, aka alpha renaming). In the future we may want to introduce further aspects of normalization that can be tested here.